sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/utils.py

```diff
@@ -223,6 +224,47 @@ def _split(_fields):
     return all_fields
 
 
+def _update_io_from_mldesigner(annotations: dict) -> dict:
+    """This function will translate IOBase from mldesigner package to azure.ml.entities.Input/Output.
+
+    This function depend on `mldesigner._input_output._IOBase._to_io_entity_args_dict` to translate Input/Output
+    instance annotations to IO entities.
+    This function depend on class names of `mldesigner._input_output` to translate Input/Output class annotations
+    to IO entities.
+    """
+    from azure.ai.ml import Input, Output
+    mldesigner_pkg = "mldesigner"
+
+    def _is_input_or_output_type(io: type, type_str: str):
+        """Return true if type name contains type_str"""
+        if isinstance(io, type) and io.__module__.startswith(mldesigner_pkg):
+            if type_str in io.__name__:
```

Comment:
Is there a specific reason that we use keyword "in" instead of strictly comparing two string to be equal?